lib/embulk/input/google_adwords.rb

```diff
@@ -131,6 +131,8 @@ def query_report_results(query, &block)
             block.call row.split(",")
           end
         end
+
+        block.call last_line.chomp.split(",")
```

Comment:
check existence of last_line

Reply:
in my thought, it doesn't matter whether last_line exists

Reply:
you mean last_line value always exists?

Reply:
```
irb(main):002:0> "".chomp.split(",")
=> []
irb(main):003:0> "hoge,fuga\n".chomp.split(",")
=> ["hoge", "fuga"]
```
if last_line is blank( "" ), no errors occur

Reply:
In that case `block.call` receives empty array and process it.
Is it OK?